demos/storybook/stories/scorecard/with-score-badge.stories.ts

```diff
@@ -0,0 +1,63 @@
+import { number } from '@storybook/addon-knobs';
+import * as Colors from '@pxblue/colors';
+import { action } from '@storybook/addon-actions';
+import { withCustomHeaderStyles } from './with-custom-header.stories';
+import { demoActions } from './with-actions.stories';
+import { ViewEncapsulation } from '@angular/core';
+
+export const withScoreBadge = (): any => ({
+    styles: [
+        `${withCustomHeaderStyles}
+        .sb-scorecard-content mat-list-item {
+            height: 36px!important;
```

Comment:
are we anticipating users having to use a lot of `!important` rules to customize our components?

Reply:
This example you've pointed out is styling the material list item that is passed into our component.   Our components will have the same limitation if the user does not use enough detail in their css selectors. 

`.red-scorecard {
    .pxb-root .pxb-header {
        background-color: map-get($pxb-red, 500);
    }
}` 

This is an example of the showcase app overwriting the default header background color.  No !important needed.